test/chi/js/modal.pug

```diff
@@ -0,0 +1,244 @@
+---
+title: Modal
+---
+script(src='/js/chi.js')
+
+// Trigger
+button#modal-trigger-1.chi-button.chi-modal__trigger(data-target='#modal-1')
+  | Launch demo modal
+// Modal
+.chi-backdrop.-closed
+  .chi-backdrop__wrapper
+    section#modal-1.chi-modal(role='dialog', aria-label='Modal description', aria-modal='true')
+      header.chi-modal__header
+        h2.chi-modal__title Modal Title
+        button.chi-button.-icon.-close(data-dismiss='modal', aria-label='Close')
+          .chi-button__content
+            i.chi-icon.icon-x
+      .chi-modal__content
+        p.-text.-m--0 Modal content
+      footer.chi-modal__footer
+        button.chi-button(data-dismiss='modal') Cancel
+        button.chi-button.-primary Save
+// JavaScript
+script.
+  chi.modal(document.getElementById('modal-trigger-1'));
```

Comment:
Let's move all the JS to a single `script.` on the bottom of the pug file to have them together.

Reply:
Fixed in the latest commit